velox/connectors/hive/HiveDataSource.cpp

```diff
@@ -101,12 +101,15 @@ HiveDataSource::HiveDataSource(
     filters.emplace(k.clone(), v->clone());
   }
   double sampleRate = 1;
-  auto remainingFilter = extractFiltersFromRemainingFilter(
-      hiveTableHandle_->remainingFilter(),
-      expressionEvaluator_,
-      false,
-      filters,
-      sampleRate);
+  auto remainingFilter = hiveTableHandle_->remainingFilter();
```

Comment:
Curious why do we need this?  `isFilterPushdownEnabled` should be ignored in Prestissimo

Reply:
It is just a workaround to get test work. Without this change, filter pushdown happens and below exception is observed.

> C++ exception with description "Exception: VeloxUserError
Error Source: USER
Error Code: UNSUPPORTED
Reason: TimestampRange: [-292275055-05-16T16:47:04.000000000, 2000-09-12T22:36:28.999999999] no nulls: testInt128() is not supported.

Reply:
This flag will be removed in the future.  Will it work if we just implement `TimestampRange::testInt128`?

Reply:
Makes sense. I will take a look. Thanks for the suggestion.

Reply:
Implemented TimestampRange::testInt128 and removed the isFilterPushdownEnabled flag.